business-registry-dashboard/app/pages/incorporateNow.vue

```diff
@@ -114,8 +114,8 @@ onMounted(async () => {
     // All errors are handled with a generic error modal
     console.error('Error in incorporate flow:', error)
     brdModal.openMagicLinkModal(
-      t('error.magicLinkGenericError.title'),
-      t('error.magicLinkGenericError.description')
+      t('error.magicLinkIncorporateNowError.title'),
```

Comment:
Just a small unrelated bug I stumbled upon.

Reply:
The comment on line 114 is now obsolete.